### PR TITLE
Make content fragment path for WF mappings configurable

### PR DESCRIPTION
**Pull request by mdickson-adbe**
Allow authors to provide mappings for the paths to content fragments that we use to derive friendly labels for Workfront values.

- Before: https://main--assets-distribution-portal--adobe.hlx.page/sample-public-site
- After: https://assets-98991--adobe-gmo--hlxsites.hlx.page/drafts/mdickson/programs

### CODE REVIEW

**Comment by shiv-gup** [comment]
@mdickson-adbe can you look into this below error, when user selects business-line then product list should be refreshed.
image

**Comment by mdickson-adbe** [comment]
Bug in the header refresh should now be resolved @shiv-gup

File: scripts/shared-program.js [hunk 1]
```diff
@@ -1,18 +1,11 @@
-import { graphqlQueryNameList } from "./graphql.js";
-import { getProductIconMapping, getBaseConfigPath } from './site-config.js';
+//import { executeQuery, graphqlQueryNameList } from "./graphql.js";
```
Comment (shiv-gup) [inline scripts/shared-program.js:1]: @mdickson-adbe can you remove this commented code

**Review by shiv-gup — APPROVED** [review]

File: scripts/shared-program.js (hunk 1 above)
Comment (mdickson-adbe) [reply scripts/shared-program.js]: Done!

**Pull request merged**